index.js

```diff
@@ -212,11 +215,22 @@ exports.writeMainView = function (dir, cb) {
 
     function view (state, emit) {
       if (state.title !== TITLE) emit(state.events.DOMTITLECHANGE, TITLE)
+
+      function handleClick () {
```

Comment:
I believe in most examples we're using function hoisting; maybe it would make sense to declare this function under the `html` block?

Reply:
Can do! :+1: